app/src/renderer/components/staking/PageValidator.vue

```diff
@@ -91,9 +160,9 @@ export default {
 <style lang="stylus">
 @media screen and (min-width: 640px)
   #validator-profile .tm-part-main
-    display flex
-    flex-flow row-reverse nowrap
+    display: flex
```

Comment:
Is your linter working?

Reply:
That was my editor automatically adding its opinion to Stylus formatting.  I see now that we're not using colons so I've changed it's settings.

Reply:
Aww!  Now I want to give you colons!